pkgs/by-name/py/pytr/package.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+python3.pkgs.buildPythonApplication rec {
```

```suggestion
python3Packages.buildPythonApplication {
```
this `rec` also goes unused